tests/test_observables.py

```diff
@@ -505,6 +514,18 @@ def test_robtex_fdns(self, mock_get=None, mock_post=None):
         ).start()
         self.assertEqual(report.get("success", False), True)
 
+    @mock_connections(patch("requests.get", side_effect=mocked_triage_get))
+    @mock_connections(patch("requests.post", side_effect=mocked_triage_post))
+    def test_triage_search(self, *args):
```

Comment:
should duplicate this test for the case `test_triage_submit` passing `"{ analysis_type": "submit" }`